noxfile.py

```diff
@@ -23,7 +23,7 @@ def test(session):
     session.install('.', 'importlib_metadata')
     session.install('pytest', 'pytest-cov', 'pytest-mock')
 
-    session.run('pytest', '--cov',
+    session.run('pytest', '--cov=src',
```

Comment:
Can we do this on a config instead?

Reply:
Considering we'll replace this with tox I'd rather do it there.

Reply:
Why? As I explained, tox is not desirable for distro packagers at the moment. This configuration is something we want to always have, and as such makes more sense to be in a config file rather than passed as an argument.

Reply:
I don't think that adds any value considering the coverage will migrate to be driven by tox, so no, not for now.

Reply:
Because wasn't before this PR so out of scope.

Reply:
But tox can make use of coverage configuration file, no? If not, why?

Reply:
> Because wasn't before this PR so out of scope.

It is not out of scope, I am asking you to add a configuration introduced by this PR in a config file instead of hardcoding it in nox/tox.

Same thing we do for flake8, mypy and isort.